### PR TITLE
WIP: Resource Form Time Periods

### PR DESCRIPTION
**Pull request by dvainio**
Wire up time period and days formsets with template.
Add fix for saving images (id and name were missing from template code).

### CODE REVIEW

File: respa_admin/templates/common/_period_day.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        {{ day.closed }}
+
+{#        <button class="btn copy-next" type="button">#}
+{#            <span class="glyphicons glyphicons-more-windows icon-left" aria-hidden="true"></span>#}
+{#            Kopioi seuravaan#}
+{#        </button>#}
+{##}
+{#        <button class="btn add-time left-group" type="button" id="btn-add-new-day">#}
+{#            <span class="glyphicon glyphicon-plus-sign icon-left" aria-hidden="true"></span>Lisää toinen aika#}
+{#        </button>#}
```
Comment (JsseL) [inline respa_admin/templates/common/_period_day.html]: Avoid having commented out a.k.a "dead code" in the PRs.

File: respa_admin/templates/resources/create_resource.html [hunk 2]
```diff
@@ -15,31 +15,15 @@
...
-                {{ period_form.as_p }}
-
-                {% if period_form.days %}
-                    {{ period_form.days.management_form }}
-                    {% for day in period_form.days %}
-                        {{ day.as_p }}
-                    {% endfor %}
-                {% endif %}
-            {% endfor %}
+{#            {% include "resources/form/_opening_hours.html" %}#}
```
Comment (JsseL) [inline respa_admin/templates/resources/create_resource.html]: Dead code

File: respa_admin/templates/common/_period_day.html (hunk 1 above)
Comment (suutari-ai) [reply respa_admin/templates/common/_period_day.html]: Actually dead code means code that is uncommented but never called. Anyway, both are bad dead code and commented code.

Comment (suutari-ai) [reply respa_admin/templates/common/_period_day.html]: Hmm.. seems that I was wrong too. :smile: According to Wikipedia [dead code](https://en.wikipedia.org/wiki/Dead_code) means code that is executed, but whose results are never used. Correct term for code that is never called is actually [Unreachable code](https://en.wikipedia.org/wiki/Unreachable_code).

**Comment by suutari-ai** [comment]
As @JsseL pointed out, there's some commented code which should be removed. Also tests would have been nice, but overall very nice work. Good job, Daniel! :+1:

I will merge this now to be able to demonstrate it in `respa-admin` branch, but I expect the commented code to be removed in another PR.

**Pull request merged**